python/wpt/__init__.py

```diff
@@ -33,6 +33,8 @@ def create_parser():
                         help="Pass preferences to servo")
     parser.add_argument('--legacy-layout', '--layout-2013', '--with-layout-2013', default=False,
                         action="store_true", help="Use expected results for the legacy layout engine")
+    parser.add_argument('--taffy-layout', default=False,
+                        action="store_true", help="Use expected results for the taffy layout engine")
```

Comment:
Instead of a command-line argument, please just use the preference. You can make it so that the preference automatically turns on grid layout and flex layout so only one needs to be passed to the command.

Reply:
Hmm... I *can* do this. But I'm reluctant too as the CLI argument seems both cleaner from a code maintenance perspective and much easier to use from a user perspective (the commands for running WPT are already really long if you need to specify multiple sub-suites to run, a report output location, etc). This also seems like it would then be different to how the legacy layout engine is specified (`--legacy-layout`), so perhaps that ought to be changed to use just a pref too if I change it for taffy layout?

Reply:
done!